### PR TITLE
Admin Log update

### PR DESCRIPTION
**Pull request by FlattestGuitar**
Full list of changes. Not in the changelog, because it’s all backstage:

- Added emptying and equipping pockets to the attack log.
- The Antag mark on logs will now be more clear allowing admins to tell much more easily whether a party is an antag.
- The following do not print to the admin’s chat even if they have attack logs on anymore:
  - Attacking with 0 force items
  - Throwing 0 throwforce items at people
  - Shooting with 0 damage projectiles
  - Equipping and stripping non-SSD players with/of items
  - Pushing and disarming people with hands

Everything that used to be saved in a mob’s attack log is STILL SAVED THERE. You can ALWAYS look things up in the attack log. The only thing this PR takes out is the awful attack log spam we get when there’s a clown with a bike horn. Why nobody fixed this earlier is beyond me.

All other diffs in there are just making things match the new system, but don’t change the functionality.

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
Now watch Travis fail this one.

**Comment by FreeStylaLT** [comment]
It didn't. 🎉

**Comment by Fox-McCloud** [comment]
>Equipping and stripping non-SSD players with/of items
Pushing and disarming people with hands

These two seem a bit iffy, especially the last one, buttttt this is more something admins should comment on. @FreeStylaLT

**Comment by FreeStylaLT** [comment]
After some discussion, it seems that both are fine. 

Disarming getting anymore serious can be ahelped, same with any high-risk items being stolen off of people.

File: code/__HELPERS/mobs.dm [hunk 1]
```diff
@@ -259,10 +259,10 @@ proc/add_logs(mob/user, mob/target, what_done, var/object=null, var/addition=nul
 		target.attack_log += text("\[[time_stamp()]\] <font color='orange'>Has been [what_done] by [key_name(user)][object ? " with [object]" : " "][addition]</font>")
 	if(admin)
 		log_attack("<font color='red'>[key_name(user)] [what_done] [key_name(target)][object ? " with [object]" : " "][addition]</font>")
-	if(istype(target) && (target.client || target.player_logged))
+	if(istype(target) && (target.key))
 		if(what_done in ignore) return
-		if(target == user)return
-		if(!admin) return
+		if(target == user) return
+		if(!print_attack_log) return
```
Comment (Fox-McCloud) [inline code/__HELPERS/mobs.dm]: Please properly put these `return`s on a new line.

**Comment by Crazylemon64** [comment]
LGTM yaya

**Pull request merged**